doc/tasks/progpilot.md

```diff
@@ -0,0 +1,40 @@
+# Progpilot
+
+The Progpilot task will run your automated PHP tasks.
+
+***Composer***
+
+```
+composer config minimum-stability dev
```

Comment:
Shouldn't we propose a released version instead?

Reply:
Unfortunately, progpilot relies on ircmaxell/php-cfg which doesn't currently have a stable version